src/Transform/ONNX/SimplifyShapeRelatedOps.cpp

```diff
@@ -60,6 +60,7 @@ Now, it's straightforward to update the output shape of Reshape from
 #include "mlir/Transforms/Passes.h"
 #include "llvm/Support/Debug.h"
 
+#include "src/Compiler/CompilerOptions.hpp"
```

Comment:
please don't add this dependency on src/Compiler from this pass

Reply:
Maybe we should have a discussion... if we can make allowances for `ONNXOpTransformPass`, I do not see any harm with doing the same thing for the c++ file above.

Reply:
the dependency in ONNXOpTransformPass is just an inconsistency and can be removed

please see PR #2506